<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>erfherder</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="containerTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <property name="icon">
      <iconset theme="document-open-recent"/>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuResources">
    <property name="title">
     <string>Resources</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionMerge"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionExport_All"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuResources"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExport_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export All</string>
   </property>
  </action>
  <action name="actionMerge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Merge</string>
   </property>
  </action>
  <action name="actionDescription">
   <property name="text">
    <string>Description</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionAbout_Qt_2">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDelete_2">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
